src/graphql/contact-details.graphql.ts

```diff
@@ -3,6 +3,7 @@ import gql from 'graphql-tag';
 export const NEWBIE_CONTACT_DETAILS = gql`
   query getContactDetails($newbieId: ID!) {
     newbie(newbieId: $newbieId) {
+      id
```

Comment:
Please fix the mocks to remove missing fields warnings

<img width="536" alt="Screenshot 2019-12-13 at 12 46 47" src="https://user-images.githubusercontent.com/12997200/70801439-ced40700-1da6-11ea-92dd-965287907e9b.png">